webapp/app.py

```diff
@@ -487,6 +487,105 @@ def takeover_index():
     view_func=engage_thank_you(engage_pages),
 )
 
+# Core docs
+core_docs = Docs(
+    parser=DocParser(
+        api=discourse_api, index_topic_id=19764, url_prefix="/core/docs"
+    ),
+    document_template="/templates/docs/discourse.html",
+    url_prefix="/core/docs",
+    blueprint_name="core",
+)
+core_docs.init_app(app)
+# Core docs - Modem Manager
+core_modemmanager_docs = Docs(
```

Comment:
I think `modemmanager` should be 2 words:

```suggestion
core_modem_manager_docs = Docs(
```